tools/junitreport/pkg/parser/gotest/parser_flat_test.go

```diff
@@ -6,6 +6,8 @@ import (
 	"reflect"
 	"testing"
 
+	"k8s.io/apimachinery/pkg/util/diff"
```

Comment:
Please let's not do this

Reply:
diff.ObjectReflectDiff is the most useful piece of code I have ever written.